site/src/app/v2/pro/ressources/page.tsx

```diff
@@ -15,7 +15,7 @@ export default function ToutSavoirSurLePassSport() {
   return (
     <>
       <PageHeader title="Ressources" subtitle="" isProVersion />
-      <main className={styles.container} tabIndex={-1} id={SKIP_LINKS_ID.mainContent}>
+      <main className={styles.container} tabIndex={-1} id={SKIP_LINKS_ID.mainContent} role="main">
```

Comment:
[question] why do you force this `role=main` inside a `main` tag ?

It looks like it's not necessary as describe in [mdn documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/main_role#prefer_html)
> Using the \<main\> element will automatically communicate a section has a role of main.

Reply:
Yes it doesn't seem necessary to me as well, but i already brought this concern up to our product designer. It seems to be only used for legacy reasons 🤷 @LeoMouyna

Reply:
Thanks for those explanations 🙏